src/clientManager.js

```diff
@@ -20,6 +20,19 @@ ClientManager.prototype.findClient = function(name) {
 	return false;
 };
 
+ClientManager.prototype.writeUser = function(userDict) {
+	var users_path = Helper.HOME + "/users" ;
+	var user_path = users_path + "/" + userDict.user + ".json";
+
+	mkdirp(users_path);
```

Comment:
I don't think so. I'd stick to UNIX practices : user list can be viewed or not : responsibility of system/user conf  (see /etc/passwd), but password are sensitive, so prevent world-reading (see /etc/shadow).